packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -137,6 +168,39 @@ export const useArbTokenBridge = (
 
   const [l11NetworkID, setL1NetWorkID] = useState<string | null>(null)
 
+  async function getL1TokenData(erc20L1Address: string) {
+    if (typeof l1Signer.provider === 'undefined') {
+      throw new Error(`No instance of L1Provider found`)
+    }
+
+    if (typeof erc20Bridger === 'undefined') {
+      throw new Error(`No instance of Erc20Bridger found`)
+    }
+
+    const l1GatewayAddress = await erc20Bridger.getL1GatewayAddress(
+      erc20L1Address,
+      l1Signer.provider
+    )
+
+    const contract = ERC20__factory.connect(erc20L1Address, l1Signer)
+
+    const multiCaller = new MultiCaller(
+      l1Signer.provider,
+      // TODO: fix, currently hard-coded to rinkeby l1 multicall
+      '0x5ba1e12693dc8f9c48aad8770482f4739beed696'
+    )
+
+    const [tokenData] = await multiCaller.getTokenData([erc20L1Address], {
+      name: true,
+      symbol: true,
```

Comment:
hmm not all erc20 tokens used here have symbols.
it is actually an optional value in the [standard](https://eips.ethereum.org/EIPS/eip-20). does this codepath still work if calling `symbol` reverts?

Reply:
ah we use [tryAggregate](https://github.com/makerdao/multicall/blob/1e1b44362640820bef92d0ccf5eeee25d9b41474/src/Multicall2.sol#L55-L66) in [multicall](https://github.com/OffchainLabs/arbitrum-sdk/blob/34b81ec289aadd8967b67c366508a718d000671a/src/lib/utils/multicall.ts#L227) so the call won't revert. but it can still return undefined